repos/spack_repo/builtin/build_systems/oneapi.py

```diff
@@ -181,7 +181,7 @@ def symlink_dir(self, src, dest):
                 link_tree = LinkTree(src_path)
                 link_tree.merge(dest_path)
             else:
-                os.symlink(src_path, dest_path)
+                symlink(src_path, dest_path)
```

Comment:
This is not imported. Next time, please do a pass yourself instead of assuming reviewers will do that for you.